tests/ui/deriving/skip/derive-skip-invalid.rs

```diff
@@ -0,0 +1,25 @@
+#![crate_type = "lib"]
+#![feature(derive_skip)]
+
+#[derive(Debug)]
+struct KeyVal(#[skip = "Debug"] usize);
+//~^ ERROR incorrect usage of the `#[skip]` attribute
+//~| NOTE the `#[skip]` attribute accepts an optional list of traits
+
+#[derive(Debug)]
+struct BadArg(#[skip("Debug")] usize);
+//~^ ERROR incorrect usage of the `#[skip]` attribute
+//~| NOTE the `#[skip]` attribute accepts an optional list of traits
+
+// FIXME: better error for derives not supporting `skip`
+// #[derive(Clone)]
+// struct SkipClone(#[skip] usize);
```

Comment:
If I understand the code correctly, we don't reject `#[skip]` (`SkippedDerives::All`) on unsupported traits at all contrary to `#[skip(...)]` (`SkippedDerives::List`) and end up generating non-sense code, right?